products/template/metadata.yml

```diff
@@ -40,8 +40,15 @@ dataset_package:
     - name: primary_csv
       filename: templatedb.csv
       type: csv
-    - name: primary_shapefile
-      filename: template.shp.zip
+    - name: shapefile_points
+      filename: templatedb_points.shp.zip
+      type: shapefile
+      overrides:
+        columns:
+          wkb_geometry:
+            name: the_geom
```

Comment:
noting that things like this column name override aren't being tested. that's what the GIS automation/metadata work is for

Reply:
ideally column names are consistent across file types, but the rationale for the renaming here is:
- shapefiles have a character limit for column names, so we either end up with unreadable auto-truncated column names or we design with the flexibility of setting them ourselves
- legacy reasons: we often want to improve column names in our "internal" tables without disrupting downstream destinations like Open Data
- since Template DB is being used to test our pipelines, this column renaming is really just to mimic an unfortunate aspect of real data products